rbac/management/consumers.py

```diff
@@ -0,0 +1,38 @@
+"""Consumers for rbac."""
+import json
+from base64 import b64decode
+
+from channels.generic.websocket import AsyncWebsocketConsumer
+
+
+class RbacConsumer(AsyncWebsocketConsumer):
+    """Consumer for websockets in rbac app."""
+
+    @staticmethod
+    def channel_group_name(account_number):
+        """Return the channel group name for an account."""
+        return f"account_{account_number}"
+
+    async def connect(self):
+        """Establish a websocket connection."""
+        headers = self.scope.get("headers", [])
+        identity_header = next((val for key, val in headers if key.decode("utf-8") == "x-rh-identity"), {})
+        decoded_identity_header = b64decode(identity_header)
```

Comment:
When I try to connect, it throws this complain because it is empty {}:
TypeError: argument should be a bytes-like object or ASCII string, not 'dict'

Looks like the connection does not go through middleware? Wondering if we should catch this error when request without identity header is made.